hugegraph-core/src/main/java/com/baidu/hugegraph/job/algorithm/AbstractAlgorithm.java

```diff
@@ -382,9 +381,7 @@ protected Iterator<Vertex> vertices(Object label, long limit) {
             ConditionQuery query = new ConditionQuery(HugeType.VERTEX);
             query.capacity(Query.NO_CAPACITY);
             query.limit(limit);
-            if (label != null) {
-                query.eq(HugeKeys.LABEL, this.getVertexLabelId(label));
-            }
+            query.eq(HugeKeys.LABEL, this.getVertexLabelId(label));
```

Comment:
keep `if (label != null)` statement to avoid label=null condition

Reply:
the `null` condition has checked in line `378` already (start of the method)